tools/async_to_sync.py

```diff
@@ -83,27 +84,31 @@ def main() -> int:
             PYVER,
         )
 
-    outputs = []
-    for fpin in opt.inputs:
-        fpout = fpin.parent / fpin.name.replace("_async", "")
-        outputs.append(str(fpout))
-        logger.info("converting %s", fpin)
-        with fpin.open() as f:
-            source = f.read()
+    with ProcessPoolExecutor(max_workers=opt.max_workers) as executor:
+        outputs = executor.map(convert, opt.inputs)
```

Comment:
Should we implement the stat check on the date, maybe it would be faster to do it outside the pool, to filter down the number of files to process, rather than in a fork? Otherwise we would fork the process just for the stat call.